test/models/normalize_eds_test.rb

```diff
@@ -56,6 +56,19 @@ class NormalizeEdsTest < ActiveSupport::TestCase
     end
   end
 
+  test 'can handle many authors' do
```

Comment:
This test is not really needed as the code you are adding operates on Results and thus doesn't care about how the authors were originally added to a Result in the first place.